xpring-eng/testnet-faucet
# Feat: GCP Deployment

## High Level Overview of Change
- Deploys faucet through Gitlab and on GCP

### Context of Change
We are migrating all of our services to GCP and Gitlab deployments. This will make it easier for us to maintain and deploy.

### Type of Change

## Before / After
This now deploys on Gitlab+GCP

## Test Plan
1. Make a commit
2. Check corresponding gitlab (note: commits do not automatically kick off a pipeline jobs)
